internal/configs/virtualserver_test.go

```diff
@@ -1247,6 +1247,10 @@ func TestCreateUpstreamServersForPlus(t *testing.T) {
 						Path:  "/coffee",
 						Route: "default/coffee",
 					},
+					{
+						Path:  "/sub",
```

Comment:
while it works, the Route that you have here is invalid, because there is no `default/sub` VSR.

Consider the following changes to make the input valid:
1. Remove the `/sub` route completely in the VS.
2. Change the `/sub` subroute in the VSR from `/sub` to `/coffee/sub`.
Both changes will make the VS and VSR valid.